packages/relay/src/lib/eth.ts

```diff
@@ -1553,17 +1561,22 @@ export class EthImpl implements Eth {
    */
   async sendRawTransaction(transaction: string, requestDetails: RequestDetails): Promise<string | JsonRpcError> {
     const requestIdPrefix = requestDetails.formattedRequestId;
-    if (transaction?.length >= constants.FUNCTION_SELECTOR_CHAR_LENGTH)
-      this.ethExecutionsCounter
-        .labels(EthImpl.ethSendRawTransaction, transaction.substring(0, constants.FUNCTION_SELECTOR_CHAR_LENGTH))
-        .inc();
-
     const networkGasPriceInWeiBars = Utils.addPercentageBufferToGasPrice(
       await this.getFeeWeibars(EthImpl.ethGasPrice, requestDetails),
     );
-
     const parsedTx = await this.parseRawTxAndPrecheck(transaction, requestDetails, networkGasPriceInWeiBars);
     const originalCallerAddress = parsedTx.from?.toString() || '';
+    const toAddress = parsedTx.to?.toString() || '';
+
+    this.ethExecutionsCounter
+      .labels(
+        EthImpl.ethSendRawTransaction,
+        transaction.substring(0, constants.FUNCTION_SELECTOR_CHAR_LENGTH) || '',
```

Comment:
Hmm, then this may not apply directly to `sendRawTransaction`. The `transaction` parameter in `sendRawTransaction` isn’t encoded the same way as in `eth_call`. `transaction` for sendRawTransaction would not have the initial 4-byte hash segment indicating the function selector. Therefore in this case, the `transaction.substring(0, constants.FUNCTION_SELECTOR_CHAR_LENGTH) || '',` would not yield any useful information as the result would not be a 4 byte transaction selector. 

However, this is not in the scope of this PR so please create a ticket to capture this and it can be address in the next PR.